airframe-config/src/main/scala/wvlet/config/YamlReader.scala

```diff
@@ -77,6 +77,10 @@ object YamlReader extends LogSupport {
             for (a <- al.asScala) {
               builder.set(k.toString, a)
             }
+          case map: java.util.Map[_, _] =>
```

Comment:
@Lewuathe This logic is a bit ad-hoc solution. What I meant in the previous comment a month ago is removing these mappings (for java arrays, maps) to ObjectBuilder side.